### PR TITLE
Support Products.PrintingMailHost.

### PR DESCRIPTION
**Pull request by jaroel**
Implements #228.
No idea on how to test this.

### CODE REVIEW

**Comment by gforcada** [comment]
For testing, injecting a global ``PRINTINGMAILHOST_ENABLED`` before calling ``get`` could be enough?

**Comment by jaroel** [comment]
@gforcada yes, it does :) Thanks!

File: src/plone/api/tests/test_portal.py [hunk 1]
```diff
@@ -302,6 +302,35 @@ def test_send_email_with_config_in_registry(self):
         msg = message_from_string(self.mailhost.messages[0])
         self.assertEqual(msg['From'], 'Registry <reg@example.org>')
 
+    def test_send_email_with_printingmailhost(self):
+        """
+        """
```
Comment (gforcada) [inline src/plone/api/tests/test_portal.py]: An empty docstring is quite useless, we could just remove it.

**Comment by jaroel** [comment]
This should just fine, could someone merge this?

**Pull request merged**